include/openthread/thread.h

```diff
@@ -1158,6 +1158,25 @@ otError otThreadDetachGracefully(otInstance *aInstance, otDetachGracefullyCallba
  */
 void otConvertDurationInSecondsToString(uint32_t aDuration, char *aBuffer, uint16_t aSize);
 
+/**
+ * Sets the store frame counter ahead.
+ *
```

Comment:
May help to document what the "frame counter ahead" parameter is to help others reading the API to underatand why/how to use it and that is should be generally configued using the `OPENTHREAD_CONFIG_STORE_FRAME_COUNTER_AHEAD`

Maybe something like:

The OpenThread stack stores the MLE and MAC security frame counter values in non-volatile storage, ensuring they persist across device resets. These saved values are set to be ahead of their current values by the "frame counter ahead" value.

Reply:
Good point, thanks.